docs/guide/installation/mac.rst

```diff
@@ -24,10 +24,10 @@ install your own version of Python. You will then have two versions of Python li
 your system at the same time. It can be confusing to make sure that when you install
 packages they are installed for the correct Python so beware.
 
-Python 2.7.3+
-^^^^^^^^^^^^^
+Python 3.6+
+^^^^^^^^^^^
 Download and install the latest version of
-`32 bit Python 2.7.3 <https://www.python.org/download/releases/2.7.3/>`
+`32 bit Python 3 <https://www.python.org/download/mac-osx/>`
```

Comment:
Seriously tho, should we not tell them to download the 64 bit one?

Reply:
`macOS 64-bit/32-bit installer`?

Reply:
or `macOS 64-bit installer`

Reply:
The latter is for most up to date systems.

Reply:
I don't know, this whole install page is criminally out of date imo.